core/vibe/core/drivers/libevent2.d

```diff
@@ -389,8 +389,29 @@ final class Libevent2Driver : EventDriver {
 
 	Libevent2TCPListener listenTCP(ushort port, void delegate(TCPConnection conn) @safe connection_callback, string address, TCPListenOptions options)
 	{
-		auto bind_addr = resolveHost(address, AF_UNSPEC, false);
-		bind_addr.port = port;
+		NetworkAddress bind_addr;
+		version(Posix)
+		{
+			import core.sys.posix.sys.un;
+			import core.stdc.string : strcpy;
+
+			if (address[0] == '/')
```

Comment:
This disallows relative paths, is that necessary?

Reply:
No, but we couldn't simply check if it's a valid path because an IP address might also be a valid path.

@s-ludwig Any objections to using `std.path.isValidPath(address) && std.socket.getAddressInfo(address, AddressInfoFlags.NUMERICHOST).empty` here? Or is there a way to do this with `vibe.core`?

We could also just change it to `address.canFind('/')`, but then we wouldn't support Windows paths and paths below the working directory without `./`.